books/projects/quadratic-reciprocity/support/euclid.lisp

```diff
@@ -6,8 +6,8 @@
 
 ;;   (2) If p is a prime divisor of a*b, then p divides either a or b.
 
-(include-book "rtl/rel11/lib/basic" :dir :system) ;; Properties of fl and mod
-(include-book "rtl/rel11/lib/util" :dir :system)  ;; Utility macros
+(include-book "rtl/rel11/support/basic" :dir :system) ;; Properties of fl and mod
+(include-book "rtl/rel11/support/util" :dir :system)  ;; Utility macros
```

Comment:
A circular dependency will happen without this patch.
`rtl/rel11/suport/float.lisp` imports 
`project/quadratic-reciprocity/euclid.lisp`
It imports  `project/quadratic-reciprocity/support/euclid.lisp`.
It import `rtl/rel11/lib/basic.lisp`.
It imports `rtl/rel11/support/top.lisp`.
It imports `rtl/rel11/support/float.lisp`.
Loop.